src/core/common/settings.cpp

```diff
@@ -76,6 +76,16 @@ struct settingsBlock
     uint16_t reserved;
 } OT_TOOL_PACKED_END;
 
+static uint16_t getAlignLength(uint16_t length)
+{
+    while (length & 3)
```

Comment:
Sorry, I missed this the first time.  You could simplify this with: `(length + 3) & 0xfffc`